README.md

```diff
@@ -76,63 +76,43 @@ Our recomended way to start a new project with ApostropheCMS is to use the [Apos
  ```
 ##### Create a project
 ```sh
-apos create-project wonderful-project
-```
-
-##### Install dependencies
-```sh
-cd wonderful-project && npm install
-```
-##### Create an admin user add them to the admin group
-```sh
-node app.js apostrophe-users:add myUser admin
+apos create-project wonderful-project --setup
 ```
+You'll be prompted to create a password for the `admin` user.
 ##### Run
 ```sh
 node app.js
 ```
 
-Navigate to `localhost:3000/login` to login with the admin credentials you created.
+Navigate to [localhost:3000/login](localhost:3000) to login with the admin credentials you created.
```

Comment:
I think you have to properly write `http://` at least for the link part for this to work.